.editorconfig

```diff
@@ -16,6 +16,6 @@ indent_size=4
 indent_style=space
 
 [*.{sh,yml,yaml}]
-indent_size=2
+indent_size=4
```

Comment:
Can this be reset to 2 (which conforms to the community convention)?

Reply:
I think indent-2 is hard to read for people. What do you think?

If you insist, I'll revert these changes : )

Reply:
Although you changed the `.editorconfig` in the root dir, which might reflect on other YAML files under this repo, here in this PR, only one YAML file gets involved. I personally think that's okay for me and the community, since most of the time the community developers won't read or edit the CI YAML configuration.

However, I don't know what other people think about this? if in the future other YAML files may be formatted into 4-spaces indentation.